main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/codilime/contrail-windows-docker/driver"
+
+	"github.com/docker/go-plugins-helpers/network"
+	"github.com/docker/go-plugins-helpers/sdk"
+)
+
+func main() {
+	d := &driver.ContrailDriver{}
+	h := network.NewHandler(d)
+
+	config := sdk.WindowsPipeConfig{
+		// This will set permissions for Everyone user allowing him to open, write, read the pipe
+		SecurityDescriptor: "S:(ML;;NW;;;LW)D:(A;;0x12019f;;;WD)",
```

Comment:
I'm not sure what they should be, let's create an issue for this and fix in another PR